components/x-live-blog-wrapper/src/LiveEventListener.js

```diff
@@ -51,11 +25,9 @@ const listenToLiveBlogEvents = ({ liveBlogWrapperId, blogPath }) => {
 		);
 	};
 
-	// TODO: use next-live-events-api v2 routes
-	// const eventSource = new EventSource(`https://next-live-event.ft.com?eventid=${blogPath}&formatted=false`, { withCredentials: true });
-	const eventSource = new EventSource(`http://localhost:3000/events`, { withCredentials: false });
+	const eventSource = new EventSource(`https://next-live-event.ft.com/v2/liveblog/${liveBlogArticleUuid}`, { withCredentials: true });
```

Comment:
Now that this is integrating with our new architecture, I think that `liveBlogArticleUuid` should be `liveBlogPackageUuid`.

Reply:
Good catch. This is not updated.